kbdcfg.lua

```diff
@@ -26,11 +26,19 @@ function kbdcfg.switch_by_name(name)
 end
 
 function kbdcfg.switch(layout)
-    for i, current in ipairs(kbdcfg.layouts) do
+
+    if kbdcfg.remember_layout then
+      if client.focus then
+        local layout = kbdcfg.layouts[kbdcfg.current]
+        client.focus.kbd_layout = layout.name
+      end
+    else
+      for i, current in ipairs(kbdcfg.layouts) do
```

Comment:
In this code you just we just looking for index of new layout and set the value to `kbdcfg.current` variable. I think that it's better to run this loop before check of client in focus.
You are right this code is not clear enough. I wrote this code about 8 months ago and may be it is a time to make small refactoring for the code after your commit.
Tomorrow, I'll check if it will work correctly. Could you please remove the if condition on line 30 and move this loop before checking client.focus? It would be nice if you could test it on your configuration. Please, let me know if it won't work.